Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_0_7.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### EWS v2
+
+- Fixed an issue where the **from** argument would cause an error when provided to the **send-mail** command.
+- Added the **from** argument to the **reply-mail** command.
```

Comment:
```suggestion
- Fixed an issue where the *from* argument would cause an error when provided to the ***send-mail*** command.
- Added the *from* argument to the ***reply-mail*** command.
```